boards/fysetc-spider/config.cfg

```diff
@@ -51,7 +51,9 @@ sensor_type: temperature_mcu
 min_temp: 0
 max_temp: 100
 
-# spi1 is also used for SD-Card extensions!
+# Use software SPI to avoid issues with displays on EXP1&2.
 [adxl345]
-spi_bus: spi1
-cs_pin: adxl345_cs_pin
+cs_pin: PA4
```

Comment:
This should still be adxl345_cs_pin, fix the alias to point to PA4 instead

Reply:
Doh. Yes that makes more sense.  Tis changed.